internal/blobtesting/asserts.go

```diff
@@ -85,7 +86,7 @@ func AssertGetBlobNotFound(ctx context.Context, t testingT, s blob.Storage, blob
 	t.Helper()
 
 	b, err := s.GetBlob(ctx, blobID, 0, -1)
-	if err != blob.ErrBlobNotFound || b != nil {
+	if !errors.Is(err, blob.ErrBlobNotFound) || b != nil {
```

Comment:
Isn't this in the standard `errors` package since Go 1.13?
Does the linter complain about using 1.13 specific functions?

Reply:
the linter cares about "errors" package and not "github.com/pkg/errors", this was a good forcing function to standardize on the latter.